research-software-directory/RSD-as-a-service
# Support for Publications Office of the European Union

## Support for Publications Office of the European Union

Changes proposed in this pull request:

* Enable support for the Publications Office of the European Union DOI registry agent
* Add some DOIs to data generation

How to test:

* `docker compose down --volumes && docker compose build --parallel && docker compose up --scale data-generation=0`
* Login, create a software or project page and add the following mentions (test it with bulk and with single entry), the fourth one should *not* be found:
	* https://data.europa.eu/doi/10.2777/28598
	* 10.2788/50967
	* https://data.europa.eu/doi/10.2788/52504
	* https://data.europa.eu/doi/10.2812/827113
* You can search for more publications [here](https://op.europa.eu/web/general-publications/publications), but most of them are *not* in the OpenAlex database.
* Run the mentions scraper a few times: `docker compose exec scrapers java -cp /usr/myjava/scrapers.jar nl.esciencecenter.rsd.scraper.doi.MainMentions`

Closes #1071

PR Checklist:
